site/data/nextdoor.ts

```diff
@@ -1,8 +1,121 @@
-import { getOrgSocialAccounts } from "./dynamo";
+import { SocialMediaPostStatus } from "@create-disruptions-data/shared-ts/enums";
+import { getParameter, putParameter } from "@create-disruptions-data/shared-ts/utils/ssm";
+import { addSocialAccountToOrg, getOrgSocialAccounts, upsertSocialMediaPost } from "./dynamo";
+import { NEXTDOOR_AUTH_URL, NEXTDOOR_URL } from "../constants";
+import {
+    GroupIds,
+    nextdoorAgencyBoundaryResultSchema,
+    nextdoorMeSchema,
+    nextdoorTokenSchema,
+} from "../schemas/nextdoor.schema";
 import { SocialMediaAccount } from "../schemas/social-media-accounts.schema";
+import { NextdoorPost } from "../schemas/social-media.schema";
+import logger from "../utils/logger";
 
-export const getNextdoorAuthUrl = () => {
-    return "";
+export const nextdoorRedirectUri = `${process.env.DOMAIN_NAME as string}/api/nextdoor-callback`;
+
+export const getNextdoorAuthHeader = async () => {
+    const { nextdoorClientId, nextdoorClientSecret } = await getNextdoorClientIdAndSecret();
+    const key = `${nextdoorClientId}:${nextdoorClientSecret}`;
+
+    return `Basic ${Buffer.from(key).toString("base64")}`;
+};
+
+export const getNextdoorSsmKey = (orgId: string) => `/social/nextdoor/${orgId}/refresh_token`;
```

Comment:
Follow on from below comment to make ssm parameter be unique for each nextdoor account token

```suggestion
export const getNextdoorSsmKey = (orgId: string, nextdoorUserId: string) => `/social/nextdoor/${orgId}/${nextdoorUserId}-refresh_token`;
```

Reply:
done